routing/router_test.go

```diff
@@ -1025,6 +1025,76 @@ func TestIgnoreNodeAnnouncement(t *testing.T) {
 	}
 }
 
+// TestIgnoreChannelEdgePolicyForUnknownChannel checks that a router will
+// ignore a channel policy for a channel not in the graph.
+func TestIgnoreChannelEdgePolicyForUnknownChannel(t *testing.T) {
+	t.Parallel()
+
+	const startingBlockHeight = 101
+	ctx, cleanUp, err := createTestCtxFromFile(startingBlockHeight,
```

Comment:
Nit: function args should be formatted on their own line.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2549/commits/29bd22862303c36fafdbcdd7df8620b752c8c8c2